server/.gitignore

```diff
@@ -1 +1,2 @@
 target
+*~
```

Comment:
You can add this kind of stuff to your personal `.git/info/exclude` file without adding it to the main repo for people who don't use it.